### DISCUSSION TITLE
请问是否可以添加百度地图中文标注

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#339] [Q&A] [answered]
希望可以添加百度地图中文标注以便和影像图叠加

**Comment by zouyaoji** [comment, marked as answer]
请问你知道百度中文标注服务地址，或者有看到哪儿有加的例子吗？

**Reply by MangMax** [reply]
目前我是这样获取的，但是貌似有时候会很多瓦片访问不到会报错，比较影响速度。
`      <vc-layer-imagery>
      <vc-imagery-provider-baidu
        url="http://maponline{s}.bdimg.com/tile/?x={x}&y={y}&z={z}&qt=vtile&styles=sl&showtext=1&scaler=2&v=083"
        :maximum-level="18"
        :projection-transforms="{ from: 'BD09', to: 'WGS84' }"
      ></vc-imagery-provider-baidu>
      </vc-layer-imagery> `

[标注接口原文](https://blog.csdn.net/jingjing_n/article/details/127406002)

**Reply by zouyaoji** [reply]
好，空了我看看呢

**Reply by MangMax** [reply]
> 好，空了我看看呢

好的，感谢

**Reply by zouyaoji** [reply]
你好，我找了一下，没看到有单独的 中文标注图层呢？

**Reply by zouyaoji** [reply]
我只看到可以通过 showtext 控制注记是否显示或者隐藏

**Reply by zouyaoji** [reply]
![image](https://user-images.githubusercontent.com/24430116/218733943-6be88b2b-ade6-43f5-8b9e-1fa4f60a700e.png)

你看加个这个属性呢吧你满足你说的需求

**Reply by MangMax** [reply]
确实没有单独的中文标注，我现在使用的是影像图加标注的。
加上这个属性可以满足了。
